src/components/Explorer/Base/Result/RunLookup/RunLookup.tsx

```diff
@@ -32,10 +44,21 @@ export const RunLookup = ({ runId, filters }: Props) => {
     React.useEffect(() => {
         if (!runId) return
         getRunTitle(runId).then(setPageTitle)
+        callApi(runId).then((data) => {
+            setTableData(data)
+        })
+        return () => {
+            setTableData([])
+        }
     }, [runId])
 
     const LinkButtons = context.result.LinkButtons
 
+    async function callApi(runId: string) {
+        const result = await fetch(`https://api.serratus.io/palmprint/run=${runId}`)
+        return result.json()
+    }
```

Comment:
Can you place this function in [src/components/Explorer/Base/Result/SerratusApiCalls.tsx](https://github.com/serratus-bio/serratus.io/blob/f1efe5e9179b0e95bebadda488ab0f5ca54801d8/src/components/Explorer/Base/Result/SerratusApiCalls.tsx)?

There, you should be able to interpolate the URL with `baseUrl`:

https://github.com/serratus-bio/serratus.io/blob/f1efe5e9179b0e95bebadda488ab0f5ca54801d8/src/components/Explorer/Base/Result/SerratusApiCalls.tsx#L5

So it would look something like 

```ts
const palmprintUrl = `${baseUrl}/palmprint/run=${runId}`
const response = await axios.get(palmprintUrl)
```

Reply:
Done: f4ddd60